src/main/java/Calculator.java

```diff
@@ -0,0 +1,79 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    Scanner scanner = new Scanner(System.in);
+    private final String ANSI_RED = "\u001B[31m";
+    private final String ANSI_RESET = "\u001B[0m";
+    private double tempArg;
+
+    public void numbersValidator() {
```

Comment:
according to Oracle's naming convention methods should be verbs: numbersValidator -> validate(), validateInput()